example/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
-add_custom_target(examples DEPENDS examples)
+add_custom_target(examples DEPENDS examplesae)
```

Comment:
Thank you for catching this. I was fiddling with the custom target name and `DEPENDS` spec, apparently, missing to restore things to normal.

I'm not clear if `DEPENDS` is needed at all though. Thoughts?